simplemonitor/Alerters/healthchecks.py

```diff
@@ -0,0 +1,127 @@
+"""
+SimpleMonitor alerts via healthchecks
+"""
+
+from typing import cast
+
+import json
+import requests
+
+from ..Monitors.monitor import Monitor
+from .alerter import Alerter, AlertLength, AlertType, register
+from ..util import check_group_match
+
+
+@register
+class HealthchecksAlerter(Alerter):
+    """Send push notification via Healthchecks."""
+
+    alerter_type = "healthchecks"
+    default_headers = {"User-Agent": "SimpleMonitor"}
+
+    def __init__(self, config_options: dict) -> None:
+        super().__init__(config_options)
+        self.hc_token = cast(
+            str, self.get_config_option("token", required=True, allow_empty=False)
+        )
+        self.hc_create = cast(str, self.get_config_option("create", required_type=bool))
```

Comment:
`required_type` takes a str not a type :)

Reply:
Sorry, I have overlooked this.